voltdb/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.yahoo.ycsb</groupId>
+    <artifactId>binding-parent</artifactId>
+    <version>0.17.0-SNAPSHOT</version>
+    <relativePath>../binding-parent</relativePath>
+  </parent>
+
+  <artifactId>voltdb-binding</artifactId>
+  <name>VoltDB Binding</name>
+  <packaging>jar</packaging>
+  <dependencies>
+   <dependency>
+      <groupId>com.yahoo.ycsb</groupId>
+      <artifactId>core</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+     <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+<!-- https://mvnrepository.com/artifact/org.voltdb/voltdbclient -->
+<dependency>
+    <groupId>org.voltdb</groupId>
+    <artifactId>voltdbclient</artifactId>
+    <version>9.0</version>
+</dependency>
+<!-- https://mvnrepository.com/artifact/org.voltdb/voltdb -->
+<dependency>
+    <groupId>org.voltdb</groupId>
+    <artifactId>voltdb</artifactId>
```

Comment:
just curious, why this dependency if we already have the volddbclient dependency?

Reply:
"voltdb" refers to the API for writing stored procedures in side the server - see https://docs.voltdb.com/javadoc/procedure-api/ . Since we compile procedures we need it. The rest of the code uses the client API (https://docs.voltdb.com/javadoc/java-client-api/).